### DISCUSSION TITLE
有关关闭余额查询的功能

### DISCUSSION THREAD

**Discussion by ytkuo81** [GaiZhenbiao/ChuanhuChatGPT#541] [Ideas]
是否可以加一下关闭余额查询的设置？
看到 log 上的余额查询，需要使用 session key，应该也只能用浏览器登入去查。
不如直接关闭，避免频繁请求，造成后续可能的不良影响。
2023-04-04 14:17:39,415 [ERROR] [openai_func.py:82] 获取API使用情况失败:API request failed with status code 403: {
  "error": {
    "message": "Your request to GET /dashboard/billing/credit_grants must be made with a session key (that is, it can only be made from the browser). You made it with the following key type: secret.",
    "type": "server_error",
    "param": null,
    "code": null
  }
}

**Comment by GaiZhenbiao** [comment]
现在去除了这部分代码，这个接口不再可用了。
